v_6_0_0/files/k8s-resource/calico-all.yaml

```diff
@@ -286,21 +286,23 @@ metadata:
   name: calico-node
   namespace: kube-system
   labels:
+    app.kubernetes.io/name: calico-node
     giantswarm.io/service-type: managed
```

Comment:
We might also want to set the `app.kubernetes.io/version` label for managed apps. I think it would be helpful information, uncertain if it would be misleading though.

Reply:
I imagine it would be useful but people would have to change that information in multiple places which would at some point not be forgotten. Do you know if we could automate adding that label?

Reply:
ping @MarcelMue when you have some time :)